packages/internal-lib-build/configs/.eslintrc.js

```diff
@@ -29,6 +35,12 @@ module.exports = {
     rules: {
         'prettier/prettier': ['error'],
         'no-console': 'off',
-        'no-unused-vars': ['error', {ignoreRestSiblings: true}]
+        'no-unused-vars': ['error', {ignoreRestSiblings: true}],
```

Comment:
can we make it a warning for now? @vmarta 

I see a lot of this error on the stubs branch

Reply:
i pushed a commit.